packages/shared/lib/core/network/stores/node-info.store.ts

```diff
@@ -1,8 +1,12 @@
 import { INodeInfo } from '@iota/sdk/out/types'
-import { writable } from 'svelte/store'
+import { writable, derived } from 'svelte/store'
 
 export const nodeInfo = writable<INodeInfo | undefined>(undefined)
 
 export function setNodeInfo(newNodeInfo: INodeInfo | undefined): void {
     return nodeInfo.set(newNodeInfo)
 }
+
+export const nodeInfoBaseToken = derived(nodeInfo, ($nodeInfo) => $nodeInfo?.baseToken)
```

Comment:
What's the point of this store? You don't seem to be reading it from any svelte file so all the reactivity features it comes with are effectively not used at all, thus it seems more appropiate to me to just read the `nodeInfo` store